README.rst

```diff
@@ -200,11 +273,13 @@ Past contributors
 
 .. END_OF_DOCS_IMPORT
 
-.. _`getting started`: https://docs.pymedphys.com/lib/howto/get-started.html
-
 .. _`Release Notes`: ./CHANGELOG.md
 
-.. _`Library Users Guide`: https://docs.pymedphys.com/lib/index.html
-.. _`CLI Users Guide`: https://docs.pymedphys.com/cli/index.html
-.. _`Contributors Guide`: https://docs.pymedphys.com/contrib/index.html
-.. _`General`: https://docs.pymedphys.com/general/index.html
+.. _`Quick Start Guide`: https://docs.pymedphys.com/en/latest/users/get-started/quick-start.html
```

Comment:
Might we be able to get these link fixes in a separate PR and then just merge that? That way the bug fix won't be tied to the other changes in this PR

Reply:
I'd really rather not. Some of the other changes here have already been reviewed and, honestly, there really isn't much here!

It has been nearly two weeks since I replied to the last review. @sjswerdloff, are you please able to re-review asap? Otherwise, @SimonBiggs or @pchlap: perhaps one of you could instead make a call given the timeframe?